xapi-xenopsd-xc.opam

```diff
@@ -1,29 +1,20 @@
 opam-version: "1.2"
+name: "xapi-xenopsd-xc"
 maintainer: "xen-api@lists.xen.org"
 authors: "xen-api@lists.xen.org"
 homepage: "https://github.com/xapi-project/xenopsd"
-build: [
-	["./configure"]
-	[make]
-]
-install: [
- 	["./setup.bin" "-install"]
-]
-remove: [
-        ["make" "setup.data"]
-        ["./setup.bin" "-uninstall"]
-        ["ocamlfind" "remove" "xenopsd"]
-]
+dev-repo: "https://github.com/xapi-project/xenopsd.git"
+bug-reports: "https://github.com/xapi-project/xenopsd/issues"
+build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
 depends: [
-  "oasis" {build}
-  "ocamlfind" {build}
+  "jbuilder" {build}
+  "xapi-xenopsd"
   "bisect_ppx"
   "cohttp"
   "core"
   "ezxenstore"
   "fd-send-recv"
   "oclock"
```

Comment:
Where are we using oclock? Should we ditch it and replace it with `mtime` (ping @lindig)

Reply:
It's used here https://github.com/xapi-project/xenopsd/blob/master/xc/xenops_server_xen.ml#L194

Reply:
I think we should use mtime instead, like we did in xapi. oclock is not well supported and the monotonic feature may be unreliable